framework/test/unit/modules/pos/commands/register_validator.spec.ts

```diff
@@ -324,6 +328,68 @@ describe('Validator registration command', () => {
 			expect(result.status).toBe(VerifyStatus.FAIL);
 			expect(result.error?.message).toInclude('Insufficient transaction fee.');
 		});
+
+		it('should throw error if name is longer than MAX_LENGTH_NAME characters long', async () => {
+			const invalidParams = codec.encode(validatorRegistrationCommandParamsSchema, {
+				...transactionParams,
+				name: 'a'.repeat(MAX_LENGTH_NAME + 1),
+			});
+
+			const invalidTransaction = new Transaction({
+				module: 'pos',
+				command: 'registerValidator',
+				senderPublicKey: publicKey,
+				nonce: BigInt(0),
+				fee: BigInt(100000000),
+				params: invalidParams,
+				signatures: [publicKey],
+			});
+
+			const context = testing
+				.createTransactionContext({
+					transaction: invalidTransaction,
+					chainID,
+				})
+				.createCommandVerifyContext<ValidatorRegistrationParams>(
+					validatorRegistrationCommandParamsSchema,
+				);
+
+			const result = await validatorRegistrationCommand.verify(context);
+
+			expect(result.status).toBe(VerifyStatus.FAIL);
+			expect(result.error?.message).toInclude('Insufficient transaction fee.');
```

Comment:
Perhaps the message could be relevant to the error intended to be checked, something of sort, "'name' is in an unsupported format", and test setup could be changed for `invalidTransaction` fee can be increased to be at least 1_000_000_000 to land on the correct error state in concern for the test case.

Reply:
For the first part about having a relevant message, this is what is returned from the implemented functionality. As far as I know, there is `Code freeze` except for tests. So am not sure if we can update the messages now. We can create a new issue to address this.

Reply:
As for the second part about the invalid transaction, I don't follow. Would you kindly elaborate? Thanks.

Reply:
I removed the check as there is no impl. Creating a bug issue for this.